### PR TITLE
fix(docs): adjust links

### PR DESCRIPTION
**Pull request by frosso**
## Description

I took a pass at the doc links around the plugin.

### Related issue(s)

Closes https://github.com/Automattic/woocommerce-services/issues/2267

### Steps to reproduce & screenshots/GIFs

- Create an international order
- Create a label
- In the "customs" section, ensure that the documentation link next to the "HS Tariff Number" redirects to the docs page

### Checklist

- [ ] unit tests

- [x] `changelog.txt` entry added

### CODE REVIEW

**Comment by harriswong** [comment]
👀

File: client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/settings.js [hunk 1]
```diff
@@ -148,7 +148,7 @@ export const CarrierAccountSettings = ( props ) => {
 					<p className="carrier-accounts__settings-subheader-description">
 						{ translate(
 							'Set up your own UPS carrier account to compare rates and print labels from multiple carriers in WooCommerce Shipping. Learn more about adding {{a}}carrier accounts{{/a}}.',
-							{ components: { a: <a href="https://docs.woocommerce.com/document/using-your-own-ups-account-in-woocommerce-shipping/" /> } }
+							{ components: { a: <a href="https://docs.woocommerce.com/document/productid-type-permalinks/using-your-own-ups-account-in-woocommerce-shipping//" /> } }
```
Comment (harriswong) [inline client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/settings.js:151]: There is an extra `/` at the end. The link works though. :D

**Review by harriswong — APPROVED** [review]
Tested the links on settings (https://localhost/wp-admin/admin.php?page=wc-settings&tab=shipping&section=woocommerce-services-settings&carrier=UPS) and the custom tab. Works as expected. 👍  🚢 

Feel free to merge after removing the `/`.

File: client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/settings.js (hunk 1 above)
Comment (frosso) [reply client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/settings.js:151]: Whoopsie, thanks! 💯 
```suggestion
							{ components: { a: <a href="https://docs.woocommerce.com/document/productid-type-permalinks/using-your-own-ups-account-in-woocommerce-shipping/" /> } }
```

**Pull request merged**